bndtools.m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lifecycleMappingMetadata>
+	<pluginExecutions>
+		<pluginExecution>
+			<pluginExecutionFilter>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<versionRange>[3.0.0,)</versionRange>
+				<goals>
+					<goal>bnd-process</goal>
+				</goals>
+			</pluginExecutionFilter>
+			<action>
+				<configurator>
+					<id>bndtools.m2e.bndconfigurator</id>
+				</configurator>
+			</action>
+		</pluginExecution>
+	</pluginExecutions>
+</lifecycleMappingMetadata>
```

Comment:
`bndtools.m2e/src/.gitignore` is still in the commit even though the file  `bndtools.m2e/src/bndtools/m2e/BndConfigurator.java` exists which means the src folder is non-empty.